src/python_autocite/__main__.py

```diff
@@ -81,27 +81,35 @@ def main():
                     citations.append(soup_to_citation(line, soup))
                 else:
                     print("Unable to load " + str(args.url), file=sys.stderr)
+            print()
+            # Start a new line, split with dot(s)
 
     formatted_citations = []
+
     for citation in citations:
         formatted_citations.append(formatter.format(citation))
 
-    formatted_citations.sort()
+    #formatted_citations.sort()
     if (args.to_text is not False):
         with open(args.to_text, "w") as f:
             for citation in formatted_citations:
                 f.write(citation)
                 f.write("\n\n")
     else:
+        myindex=0
         for citation in formatted_citations:
             tmp=str(citation)
+            #print("For URL: "+str(citation.url))
             if("[[[AUTHORS]]]" in tmp):
+                print("For URL: "+str(citations[myindex].url))
```

Comment:
It is not working, 
error message
```
wayne@wayne-inspiron3583:~/python-autocite$ autocite --from-file url
.....
Traceback (most recent call last):
  File "/usr/local/bin/autocite", line 11, in <module>
    load_entry_point('python-autocite==0.0.1', 'console_scripts', 'autocite')()
  File "/usr/local/lib/python3.8/dist-packages/python_autocite-0.0.1-py3.8.egg/python_autocite/__main__.py", line 101, in main
AttributeError: 'str' object has no attribute 'url'

```

Code:
``` python
    else:
        for citation in formatted_citations:
            tmp=str(citation)
            if("[[[AUTHORS]]]" in tmp):
                print("For URL: "+str(citation.url))
                tmp=tmp.replace("[[[AUTHORS]]]",input("Please enter author(s) manually: "))
            if("[[[PUBLICATION DATE]]]" in tmp):
                print("For URL: "+str(citation.url))
                tmp=tmp.replace("[[[PUBLICATION DATE]]]",input("Please enter publication date manually: "))
            if("[[[TITLE]]]" in tmp):
                print("For URL: "+str(citation.url))
                tmp=tmp.replace("[[[TITLE]]]",input("Please enter the title manually: "))
            print(tmp)
```

Reply:
Ahhhh you are correct, the diff goggles got me. This loop is indeed working with strings because it's iterating through the `formatted_citations` list. So you have it right, you'll need to keep an index if you want to look up the citation object from the `citations` list.

There's probably a more elegant way to tie those together, but just using the index will work.